pandas/core/indexes/timedeltas.py

```diff
@@ -500,7 +500,33 @@ def f(x):
 
     def total_seconds(self):
         """
-        Total duration of each element expressed in seconds.
+        Return total duration of each element expressed in seconds.
+
+        Return a series with the same length and index as the original,
+        containing the length of each element expressed in seconds.
+
+        Returns
+        -------
+        s : pandas.Series
+            a series of type `float64`.
+
```

Comment:
`See Also`section is missing. Maybe it could be useful to reference [`datetime` documentation](https://docs.python.org/3/library/datetime.html#datetime.timedelta.total_seconds).

Reply:
Added.